BedrockServer.h

```diff
@@ -480,6 +495,10 @@ class BedrockServer : public SQLiteServer {
     // This records how many outstanding socket threads there are so we can wait for them to complete before exiting.
     atomic<uint64_t> _outstandingSocketThreads;
 
+    // If we hit the point where we're unable to create new socket threads, we block doing so.
+    bool _newSocketThreadsBlocked; 
```

Comment:
To follow our naming conventions:
```suggestion
    bool _shouldBlockNewSocketThreads; 
```